### PR TITLE
[docs] : Missing link and alignment fixes

### PR DESCRIPTION
**Pull request by rakeshPRaghu**
1. Missing logo link to website home
2. See also section links margin
3. Search bar losing border on focus

Signed-off-by: Rakesh PR <rakesh.pr@mayadata.io>

### CODE REVIEW

File: docs/src/components/Header/index.js [hunk 1]
```diff
@@ -13,13 +13,14 @@ export const Header = () => {
...
     const navLinks = siteConfig?.customFields?.navLinks;
     const repoUrl = siteConfig?.customFields?.externalLinks?.repoUrl;
+    const websiteUrl = siteConfig?.url;
     const { width } = useViewport();
     return(
         <div className={styles.headerOuterWrapper}>
             <div className={styles.headerInnerWrapper}>
                 <div className={styles.header}>
                     <div className={width > 480 ? styles.logoWrapper : styles.logoWrapperSmallView }>
-                        <a href={useBaseUrl('/')} target="_self">
```
Comment (rakeshPRaghu) [inline docs/src/components/Header/index.js:22]: baseUrl was taking the user to the docs page, so added the website url to go homepage

File: docs/src/scss/_typography.scss [hunk 2]
```diff
@@ -53,6 +53,10 @@ tbody {
   }
 }
 
+p a + a {
+  margin-left: 1rem;
```
Comment (rakeshPRaghu) [inline docs/src/scss/_typography.scss:57]: giving adjacent urls some space between them

Comment (IsAmrish) [reply docs/src/scss/_typography.scss:57]: should we try using classnames?

Comment (rakeshPRaghu) [reply docs/src/scss/_typography.scss:57]: its in the md file, thats why didnt gave any classnames

Comment (IsAmrish) [reply docs/src/scss/_typography.scss:57]: ok

Comment (IsAmrish) [reply docs/src/scss/_typography.scss:57]: since this rules for markdown content only,  we should apply css property using `.markdown` classname

Comment (IsAmrish) [reply docs/src/scss/_typography.scss:57]: same comments for other css rules which were applied with intend of applying on markdown content

Comment (rakeshPRaghu) [reply docs/src/scss/_typography.scss:57]: > same comments for other css rules which were applied with intend of applying on markdown content

does that mean for all the styles which come inside of `.markdown`?

Comment (IsAmrish) [reply docs/src/scss/_typography.scss:57]: yes,

I know this will be big change. :) 
If you can please do this or do it in separate PR

**Review by IsAmrish — APPROVED** [review]
LGTM

**Review by AVRahul — APPROVED** [review]

**Review by satyapriyamishra222 — APPROVED** [review]

**Pull request merged**